runtime-light/coroutine/awaitable.h

```diff
@@ -187,9 +206,11 @@ class wait_fork_t {
   void await_suspend(std::coroutine_handle<> coro) noexcept {
     fork_awaiter.await_suspend(coro);
     timer_awaiter.await_suspend(coro);
+    suspended_fork_id_ = ForkComponentContext::get().get_running_fork_id();
```

Comment:
Did you try it? You'll get multiple equal logs. There will be `3` equal log lines. Let's remove this and keep just `2` :)

Reply:
Yeah, there is a bug, when awaitable is await_ready ForkComponentContext set default int64_t{} as current running. Fix it